packages/capabilities/src/filecoin/dealer.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Filecoin Dealer Capabilities
+ *
+ * These can be imported directly with:
+ * ```js
+ * import * as Dealer from '@web3-storage/capabilities/filecoin/dealer'
+ * ```
+ *
+ * @module
+ */
+
+import { capability, Schema, ok } from '@ucanto/validator'
+import { PieceLink } from './lib.js'
+import { equalWith, checkLink, and } from '../utils.js'
+
+/**
+ * Capability allowing an Aggregator to request an aggregate to be added to a
+ * deal with a Storage Provider.
+ */
+export const aggregateOffer = capability({
+  can: 'aggregate/offer',
+  /**
+   * DID of the Aggregator.
```

Comment:
```suggestion
   * DID of an authorized Storefront.
```

Reply:
Per previous discussion, it should be Storefront who requests dealer for something to be offered. Aggregation is an implementation detail for storefront perspective.

For now in implementation, we will just use same DID for Aggregator and Storefront, but to make sure we are using the Storefront one